implementations/rust/ockam/ockam_command/src/vault/mod.rs

```diff
@@ -148,3 +148,23 @@ async fn run_impl(ctx: Context, (opts, cmd): (CommandGlobalOpts, VaultCommand))
     }
     Ok(())
 }
+
+pub fn default_vault_name() -> String {
+    let res_cli = CliState::try_default();
+
+    let cli_state = match res_cli {
```

Comment:
I think we could encapsulate such a function in `CliState`, as `cli_state.default_vault_name()` and reduce the code duplication with the lines [here](https://github.com/build-trust/ockam/pull/4360/files#diff-c2b89106900cbb8182b0b1a5281431f571738a0f5ff81df4d0850953888ae251R51)

Reply:
I will create a GH issue for this, as well with the idea of abstracting some of the CLI State configs (Node, Vault, Identity, etc).

we have other cases where it could be used too - e.g. https://github.com/build-trust/ockam/blob/da9c020b3f78906d41946d998cb852b30e61674c/implementations/rust/ockam/ockam_command/src/node/mod.rs#L87